advanced_filters/urls.py

```diff
@@ -1,14 +1,30 @@
-from django.conf.urls import url
+try:
+    from django.urls import path
+except ImportError:
+    path = None
+    from django.conf.urls import url
```

Comment:
`path` function came with Django 2.2, which is the current minimum supported version. No need for the if block.

Reply:
This PR is not relevant anymore, the develop branch already includes this fix.